README.md

```diff
@@ -22,10 +22,11 @@ When you get started with The Things Network, you'll probably have some question
 1. Make sure you have [Go](https://golang.org) installed (version 1.7 or later).
 2. Set up your [Go environment](https://golang.org/doc/code.html#GOPATH)
 3. Install the [protobuf compiler (`protoc`)](https://github.com/google/protobuf/releases)
-4. Make sure you have [Redis](http://redis.io/download) and [RabbitMQ](https://www.rabbitmq.com/download.html) **installed** and **running**.  
+4. Install `make`. On Linux install `build-essential`. On MacOS, make comes with XCode or the developer tools. On Windows you can get make from [https://gnuarmeclipse.github.io/windows-build-tools/](https://gnuarmeclipse.github.io/windows-build-tools/)
```

Comment:
"On MacOS": macOS (lowercase m) and `make` (with backticks)
"you can get":  `make` (with backticks)